librepo/yum.h

```diff
@@ -72,6 +85,72 @@ lr_yum_repo_free(LrYumRepo *repo);
 const char *
 lr_yum_repo_path(LrYumRepo *repo, const char *type);
 
+int
+hmfcb(void *clientp, const char *msg, const char *url);
```

Comment:
@cgwalters Do you know, it is already outdated and there is a comment above the function?